demo/config.xml

```diff
@@ -1,14 +1,31 @@
+/*
```

Comment:
This is suddenly NOT a valid XML file ...
See: https://github.com/apache/cordova-plugin-screen-orientation/blob/master/plugin.xml#L1 for an example of how to add the license header to XML files.